src/components/OSCALControl.js

```diff
@@ -101,16 +102,18 @@ export default function OSCALControl(props) {
       <CardContent>
         <Grid container spacing={1}>
           <Grid item xs={12}>
-            <Typography
-              variant="h6"
-              component="h2"
-              style={props.childLevel ? { fontSize: "1.1rem" } : undefined}
-            >
-              <span style={{ textTransform: "uppercase" }}>
-                {props.control.id}
-              </span>{" "}
-              {props.control.title} {modificationDisplay}
-            </Typography>
+            <OSCALAnchorLinkHeader value={props.control.id}>
```

Comment:
Could this work?

```suggestion
            <OSCALAnchorLinkHeader value={props.control.id} isControl>
```

Reply:
I like the thought, although this will not solve the issue of whether the provided fragment in the URL is a control or not. I have a feeling we will still need some sort of check that is done at a high level to determine whether the format follows a certain control scheme. Right now it just fits NIST SP-800-53 Rev 5. 

I had thought about making the fragment include "control" to help make the control check easier, although "#control-ac-1" doesn't look as clean in my opinion. Furthermore, we would need to validate the fragment isn't "control-implementation", "control-groups", or any other fragment starting with "control".